### PR TITLE
66 meta info files

### LINKED ISSUE #66: Add missing meta info files
1) For better support of internal open source tooling, we should have a file containing the maintainers. (DONE in #68)
2) CatWatch. Is this also need for incubator projects?
3) More stuff we could takle with this issue?
4) Contributing.md see #20

### PR DESCRIPTION
**Pull request by Retro64**
close #66

### CODE REVIEW

**Comment by Retro64** [comment]
:+1:

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at dag.agre@zalando.fi. All
```
Comment (bzums) [inline CODE_OF_CONDUCT.md]: there should be some other mail (if one at all), right?

Comment (Retro64) [inline CODE_OF_CONDUCT.md]: Suggestion: Link to the Maintainers-File

**Comment by Retro64** [comment]
:+1:

**Comment by bzums** [comment]
👍

**Pull request merged**